### PR TITLE
New windows onboarding

### PR DESCRIPTION
**Pull request by kpslp**

### CODE REVIEW

File: onboard-device/windows-public-officer copy.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Onboard Windows device to SEED as public officer
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-public-officer copy.md:1]: @kpslp pls check the file naming convention

File: onboard-device/windows-public-officer copy.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+> **Note:** Based on your Windows settings, you may be prompted to restart or reset your password while onboarding.
+
+
+<div style="position:relative;padding-bottom:56.25%;padding-top:30px;height:0;overflow:hidden;">
+<iframe style="position:absolute;top:0;left:0;width:100%;height:100%;"  src="https://www.youtube.com/embed/Cvb7lppxFqs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen="true"></iframe>
+</div>
+
+> **Note**: Please complete **Step 1** and **Step 2** within the same session.
+
+## Step 1: Set up Microsoft Intune
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-public-officer copy.md:14]: @kpslp  I have reverted this to "Enrol <> device using Company Portal app"

File: onboard-device/windows-public-officer copy.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+<details>
+  <summary style="font-size:18px">Enrol your device in Microsoft Intune</summary><br>
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-public-officer copy.md:17]: in macOS , i have stated it as "Set up Microsoft Intune to get the required applications and device configurations"

File: onboard-device/windows-public-officer copy.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+1. Click **Start** icon on the taskbar.
+
+2. Go to **Settings** > **Accounts** > **Access work or school** and click **Connect** plus sign to add your work or school account.
+
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-public-officer copy.md:22]: pls remove the "plus sign "

File: onboard-device/windows-public-officer copy.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+<kbd>![access-work-or-school](../images/onboarding-instructions-for-windows/access-work-or-school.png ':size=600')</kbd>
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-public-officer copy.md:23]: call out the "connect" on this image

File: onboard-device/windows-public-officer copy.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+3. Approve your TechPass login using the authenticator app.
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-public-officer copy.md:25]: is this the TechPass login? work or school account is not techpass for PO.

File: onboard-device/windows-public-officer copy.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+Authorise your WOG account by entering the verification code displayed for your SG Govt M365 profile on the authenticator app before approving your Techpass login.
+
+<kbd>![log-in-to-gcc](../images/onboarding-for-macos/log-in-to-gcc.png ':size=500')</kbd>
+
+
+<kbd>![settings](../images/onboarding-instructions-for-windows/settings.png ':size=600')</kbd>
+
+Your account is added and listed as a connection. This account has **Info** and **Disconnect** options as shown below.
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-public-officer copy.md:33]: where are these options?

File: onboard-device/windows-public-officer copy.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+Authorise your WOG account by entering the verification code displayed for your SG Govt M365 profile on the authenticator app before approving your Techpass login.
+
+<kbd>![log-in-to-gcc](../images/onboarding-for-macos/log-in-to-gcc.png ':size=500')</kbd>
+
+
+<kbd>![settings](../images/onboarding-instructions-for-windows/settings.png ':size=600')</kbd>
+
+Your account is added and listed as a connection. This account has **Info** and **Disconnect** options as shown below. 
+
+4. Select the **Info** option and verify that a similar result to the following is displayed.
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-public-officer copy.md:35]: User would like to know where is the info option?

File: onboard-device/windows-public-officer copy.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+<kbd>![managed-by-sg-govt-m365](../images/onboarding-instructions-for-windows/managed-by-sg-govt-m365.png ':size=600')</kbd>
+
+## Step 2: Register Microsoft Intune Device ID (only for GSIB users onboarding their Internet Device)
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-public-officer copy.md]: pls remove the text in brackets.

File: onboard-device/windows-vendor.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+
+<!--
+> **Note**:  
+>
+>- This step is applicable only to users with a GSIB device and their TechPass ID matches their organization's email address domain, excluding those whose TechPass ID's domain is ```techpass.gov.sg```.
+>
+>- If you only have a SE-GSIB device, complete step 1 and contact the [support team](https://go.gov.sg/techpass-sr) with your Intune Device ID to register your Intune Device ID.  
+
+- Skip the following steps if your TechPass ID belongs to the TechPass AAD and has its domain as *techpass.gov.sg*. For example, *peter_wilson<span>@</span>techpass.gov.sg*.-->
+
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-vendor.md:51]: remove lines 43-50

File: onboard-device/windows-vendor.md [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+    $enrollmentIdKey = $enrollmentsKey.OpenSubKey($name)
+    if ($enrollmentIdKey.GetValue("ProviderID") -ieq "MS DM Server") {
+        $intune_id = $enrollmentIdKey.OpenSubKey("DMClient\MS DM Server").GetValue("EntDMID", "Intune ID not found")
+        break
+    }
+}
+Write-Output $intune_id
+```
+2. Take note of the Intune Device ID that is displayed on the Powershell window.
+
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-vendor.md:70]: pls add the screenshot displaying the Intune Device ID

File: onboard-device/windows-vendor.md [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+4. Click **Onboard device to SEED** and follow the on-screen instructions to submit this Intune Device ID.
+5. Ensure that your device is connected to the Internet so that Intune is able to install the required SEED components and configurations. 
+6. Within the next 2 hours, check your inbox (organisational email address) to see if you have received the successfully onboarded email.
+7. If you don't receive this email after two hours, submit an [incident request](https://go.gov.sg/techpass-sr). 
+8. After receiving the email notification, a desktop notification will be sent to inform you of the device name change and snd requesting that you restart the device. Do the following:
+
+  i. Save your current work and restart your device.
+
+  ii. If prompted to specify your password, enter it.
+
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-vendor.md:83]: make verification steps as step 3

Comment (SenthilKumarSivakami) [inline onboard-device/windows-vendor.md:83]: are the steps mandatory to proceed with the verification?

File: onboard-device/windows-vendor.md [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+4. Select the **Info** option and verify that a similar result to the following is displayed. You will see **TechPass** instead of **SG Govt M365**.
+
+<kbd>![managed-by-sg-govt-m365](../images/onboarding-instructions-for-windows/managed-by-sg-govt-m365.png ':size=600')</kbd>
+
+>**Notes**:
+
+>- Tanium and Cloudflare WARP are installed on your device. When the installation is complete, Microsoft Intune Management Extension sends you a desktop notification.
+
+## Step 2: Register Microsoft Intune Device ID
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-vendor.md:39]: remove step 2 as it is not applicable for vendors.

File: onboard-device/windows-public-officer copy.md [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+5. Ensure that your device is connected to the Internet so that Intune is able to install the required SEED components and configurations. 
+6. Within the next 2 hours, check your inbox (organisational email address) to see if you have received the successfully onboarded email.
+7. If you don't receive this email after two hours, submit an [incident request](https://go.gov.sg/techpass-sr). 
+8. After receiving the email notification, a desktop notification will be sent to inform you of the device name change and snd requesting that you restart the device. Do the following:
+
+  i. Save your current work and restart your device.
+
+  ii. If prompted to specify your password, enter it.
+
+9. To verify if the required profiles are installed correctly, go to the Internet Device onboarded to SEED, open **Settings** > **Apps** > **Apps & features** and ensure that Tanium and Cloudflare WARP are listed.
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-public-officer copy.md]: Verification should be step 3.

File: onboard-device/windows-vendor.md [hunk 15]
```diff
@@ -0,0 +1,88 @@
+# Onboard Windows device to SEED as vendors and contractors
+
+<!-- This page is linked in the TechPass portal-Register Intune Device ID, so please do not rename this file. -->
```
Comment (SenthilKumarSivakami) [inline onboard-device/windows-vendor.md]: pls remove this comment.

File: onboard-device/windows-vendor.md [hunk 16]
```diff
@@ -0,0 +1,88 @@
...
+
+> **Note:** Based on your Windows settings, you may be prompted to restart or reset your password while onboarding.
+
+
+<div style="position:relative;padding-bottom:56.25%;padding-top:30px;height:0;overflow:hidden;">
```
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply onboard-device/windows-vendor.md:6]: Pls click Edit comment to copy the actual code this does not display the <div> settings.

File: onboard-device/windows-vendor.md (hunk 15 above)
Comment (kpslp) [reply onboard-device/windows-vendor.md]: not sure why you are sill seeing it but it's not there already.

File: onboard-device/windows-public-officer copy.md (hunk 1 above)
Comment (kpslp) [reply onboard-device/windows-public-officer copy.md]: changes made. removed previous windows public officer and now using the original windows.md

File: onboard-device/windows-public-officer copy.md (hunk 3 above)
Comment (kpslp) [reply onboard-device/windows-public-officer copy.md]: followed the same convention

File: onboard-device/windows-public-officer copy.md (hunk 4 above)
Comment (kpslp) [reply onboard-device/windows-public-officer copy.md]: done

File: onboard-device/windows-public-officer copy.md (hunk 5 above)
Comment (kpslp) [reply onboard-device/windows-public-officer copy.md]: done

File: onboard-device/windows-public-officer copy.md (hunk 6 above)
Comment (kpslp) [reply onboard-device/windows-public-officer copy.md]: updated

File: onboard-device/windows-public-officer copy.md (hunk 7 above)
Comment (kpslp) [reply onboard-device/windows-public-officer copy.md]: updated

File: onboard-device/windows-public-officer copy.md (hunk 8 above)
Comment (kpslp) [reply onboard-device/windows-public-officer copy.md]: screenshot is displayed

File: onboard-device/windows-public-officer copy.md (hunk 9 above)
Comment (kpslp) [reply onboard-device/windows-public-officer copy.md]: done

File: onboard-device/windows-vendor.md (hunk 10 above)
Comment (kpslp) [reply onboard-device/windows-vendor.md:29]: done

File: onboard-device/windows-vendor.md (hunk 11 above)
Comment (kpslp) [reply onboard-device/windows-vendor.md:31]: done

File: onboard-device/windows-vendor.md (hunk 12 above)
Comment (kpslp) [reply onboard-device/windows-vendor.md:39]: done, step 2 because vendors dont need to register

Comment (kpslp) [reply onboard-device/windows-vendor.md:39]: checked w kp and shon, removed

File: onboard-device/windows-vendor.md (hunk 13 above)
Comment (kpslp) [reply onboard-device/windows-vendor.md]: done

File: onboard-device/windows-public-officer copy.md (hunk 14 above)
Comment (kpslp) [reply onboard-device/windows-public-officer copy.md]: this is obsolete, i've deleted this md and am using windows.md

File: onboard-device/windows-vendor.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (kpslp) [reply onboard-device/windows-vendor.md:5]: did it previously, copy pasted this but noticed the content wasnt showing. checked on macos , there's div tags, added it in again

**Review by eu-tee — APPROVED** [review]
Ok, thank you Huda. But i still cant quite visualise. But ok to go first. If any changes then we tweak again.

**Pull request merged**